tests/js/client/aql/aql-zkdindex-prefix-fields.js

```diff
@@ -35,9 +35,10 @@ function optimizerRuleZkd2dIndexTestSuite() {
   let col;
 
   return {
-
     tearDown: function () {
-      db[colName].drop();
+      if (db[colName]) {
+        db[colName].drop();
+      }
```

Comment:
nit: can be simplified to just
```suggestion
      db._drop(colName);
```
`db._drop()` will not fail if no such collection exists.